src/sipmpdf/functions.py

```diff
@@ -259,4 +372,17 @@ def sipm_response(
   Using numba for calculation acceleration (the function is difficult to express
   in pure array syntax)
   """
-  pass
+  kern = kernel_switch(x, pedestal, gain, common_noise, pixel_noise, poisson_mean, poisson_borel, ap_prob, ap_beta)
+ 
+  Pdc=0 ##probability of dark current, define here, fix all these values later
+  smear_of_darkcurrent=0 ##define here
```

Comment:
sigma_0 in the paper corresponds to the `common_noise` variable defined in the function's input. 
sigma_1 in the paper corresponds to the `pixel_noise` variable defined in the function's input. 

Dark current smearing factor should be `kern.sqrt(sigma_0**2 + sigma_1**2)`